tests/UnpackerTest.php

```diff
@@ -40,6 +40,9 @@ public function testDoNotDuplicateEntry(): void
         file_put_contents($composerJsonPath, '{}');
 
         $originalEnvComposer = getenv('COMPOSER');
+        $originalEnvComposer = $_SERVER['COMPOSER'];
```

Comment:
should it be a different variable name ?

Reply:
no need to, esp for a test file

Reply:
then the previous line is dead code.